django_sorcery/field_mapping.py

```diff
@@ -180,10 +183,11 @@ def build_standard_field(self, attr, **kwargs):
                 kwargs.update(attr.field_kwargs)
                 return type_func(attr, **kwargs)
 
-        for base in attr.column.type.python_type.mro():
-            if base in FIELD_LOOKUP:
-                kwargs.update(attr.field_kwargs)
-                return FIELD_LOOKUP[base](**kwargs)
+        with suppress(NotImplementedError):
```

Comment:
example?

Reply:
Some custom types don't implement `python_type`, hence it blows up